code/modules/modular_computers/hardware/battery_module.dm

```diff
@@ -15,7 +15,6 @@
 	desc = "An advanced power cell, often used in most laptops. It is too large to be fitted into smaller devices. It's rating is 1100."
 	icon_state = "battery_advanced"
 	origin_tech = list(TECH_POWER = 2, TECH_ENGINEERING = 2)
-	hardware_size = 3
```

Comment:
Removing this is going to let you put these into tablets, no?

Reply:
fixed